README.md

```diff
@@ -14,6 +14,22 @@ If you start developing a plugin that you aim to release as open source, we sugg
 
 You can also use this process if you have an idea for a good plugin but you hope that someone else will pick up the work.
 
+## First run
+
+The Backstage has multiple dependencies. To pass through their installation, make sure to follow the [isolated-vm](https://github.com/laverdet/isolated-vm#requirements) requirements.
+
+Those instructions are missing one more:
+
+```console
+sudo dnf install libuv-devel
+```
```

Comment:
Seems like this is a distribution-specific instruction to resolve a single dependency issue. Would you mind rather sending this suggestion upstream to `isolated-vm` directly? This is not specific to our repository, it seems like a common issue to all `isolated-vm` users on Fedora/RHEL.

Reply:
ok, done: https://github.com/laverdet/isolated-vm/pull/408